src/main/java/build/buildfarm/common/redis/RedisNodeHashes.java

```diff
@@ -86,18 +86,18 @@ public static List<String> getEvenlyDistributedHashesWithPrefix(
       JedisCluster cluster = (JedisCluster) jedis;
       Iterable<List<List<Long>>> nodeSlotRanges = getNodeSlotRanges(cluster);
       try {
-        ImmutableList.Builder hashTags = ImmutableList.builder();
+        ImmutableList.Builder<String> hashTags = ImmutableList.builder();
         for (List<List<Long>> slotRanges : nodeSlotRanges) {
           // we can use any slot that is in range for the node.
           // in this case, we will use the first slot.
           hashTags.add(RedisSlotToHash.correlateRangesWithPrefix(slotRanges, prefix));
         }
         return hashTags.build();
       } catch (JedisException e) {
-        return ImmutableList.of();
+        return List.of();
       }
     } else {
-      return ImmutableList.of(prefix);
+      return List.of(prefix);
```

Comment:
I don't agree with this (or really the List response - it should be Iterable) - I don't want the response to be a mutable object.

Reply:
No worries. IntelliJ keeps recommending this refactor so I thought I'd try it. I'll just fixup this commit.